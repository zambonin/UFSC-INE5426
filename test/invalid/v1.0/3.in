bool ref b
b = addr false
