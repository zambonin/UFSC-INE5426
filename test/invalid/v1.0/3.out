[Line 2] semantic error: address operation expects a variable or array item
